docs/hooks-nodejs.md

```diff
@@ -188,6 +188,25 @@ before("Machines > Machines collection > Get Machines", function (transaction) {
   transaction.request.body = JSON.stringify(requestBody);
 });
 ```
+### Modifying Multipart Transaction Request Body Prior to Execution
+
+```javascript
+const hooks = require('hooks');
+const Multipart = require('multi-part');
+const fs = require('fs');
+const streamToString = require('stream-to-string');
+
+var before = hooks.before;
+
+before("Machines > Machines collection > POST Machines", async function (transaction, done) {
+    const form = new Multipart();
+    form.append('title', 'Paykan Jvanan Gojeii');
+    form.append('photo', fs.createReadStream('./peykan.jpg'));
```

Comment:
The title is "Paykan", but the photo is "peykan". Is it a typo? This is really just a nitpick, but it could contribute to confusion of inexperienced readers.